python/cucim/src/cucim/skimage/segmentation/_chan_vese.py

```diff
@@ -89,7 +89,7 @@ def _cv_calculate_variation(image, phi, mu, lambda1, lambda2, dt):
     similar approach is used by Rami Cohen, and it is from there that the
     C1-4 notation is taken.
     """
```

Comment:
Do we need to normalize the input `image`'s `dtype` somehow?

```suggestion
    """
    float_dtype = _supported_float_type(image.dtype)
```

Reply:
This function determines whether to use 32 or 64-bit floating point types based on [this dict](https://github.com/rapidsai/cucim/blob/61d8a3923d2f6c87da136676e64754796f379826/python/cucim/src/cucim/skimage/_shared/utils.py#L870-L887)

Reply:
Gotcha. Thanks Greg! 🙏